eng/Versions.props

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.9.0</VersionPrefix>
+    <VersionPrefix>17.10.0</VersionPrefix>
```

Comment:
It seems thtat this change comes from the merging main into feature branch. 
since the target branch is vs17.9 will it be possible to merge vs17.9 instead of main branch?

Reply:
reverted the merge from main

Reply:
Thank you for updates!